src/main/kotlin/app/FactCodes.kt

```diff
@@ -1,10 +1,19 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
 package app
 
 object FactCodes {
     val COMMITS_DAY_WEEK = 1
     val COMMITS_DAY_TIME = 2
+    val COMMITS_LINE_NUM_AVG = 8
+    val COMMITS_NUM = 9
+    // A map of line numbers to commits number. Used in a commit histogram.
+    val COMMITS_NUM_TO_LINE_NUM = 12
```

Comment:
i'm sort of concerned that commits_num has the plural form, but line_num has the single form.

Reply:
Fix